### PR TITLE
docs: added docs for meshmodel

### LINKED ISSUE #8221: [Docs] MeshModel User Guides
#### Desired State
User Guide can include:

- MeshModel Browser in Settings in Meshery UI
- mesheryctl system model commands
- Meshery Designer, which allows them to configure components
- MeshMap, which allows them to interact with and create designs with components

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Yana-Gupta**
**Notes for Reviewers**

This PR fixes #8221
The pr fixed point 1 and 4 mentioned in the issue.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8b5eea50ee3d719ec1d342b08df5f07b553c6127 at: https://64c21c21c83715557503a8ae--meshery-docs-preview.netlify.app

**Review by Boombag0607 — COMMENTED** [review]
@Yana-Gupta Looks great! Nice work. (Just make sure you stay consistent with the spellings)

**Comment by Yana-Gupta** [comment]
> @Yana-Gupta Looks great! Nice work. (Just make sure you stay consistent with the spellings)

Thank you! I'll make sure to maintain consistency with the spellings.

**Comment by l5io** [comment]
🚀 Preview for commit 345ef2c5c8f0a91cf1628f516538cbd6c071f730 at: https://64c240f1af01d46fb007dcea--meshery-docs-preview.netlify.app

**Comment by Yana-Gupta** [comment]
@thisiskaransgit and @Chadha93 would you review it once.

**Comment by sudhanshutech** [comment]
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
Hey @Yana-Gupta, thanks for working on this 👍🏼 , some things for you to consider here would be 
For Mehsmap: 
- How to access the extension's UI should be explained once 
- We have thoroughly described Mehsmap and how to use its features in /extensions section/ CTAs for the page should be included in the page.
- images in a queue does not look good

1. Include the href of layer5.io pages for respective modes.
2. Take reference from the websites pages and community resources to describe the steps
3. Include current existing gifs, on the page if it describes the step better.

For Meshmodel page, include a short paragraph describing  meshmodal summary

File: docs/assets/img/meshmap/meshmap-design-creation.png
Comment (thisiskaransgit) [inline docs/assets/img/meshmap/meshmap-design-creation.png:1]: @Yana-Gupta, not this image, please.

File: docs/pages/guides/meshmap.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+MeshMap has two modes:
+
+1. Designer
+2. Visualizer
+
+### Designer
+
+**Step 1. Access the MeshMap in Meshery UI**
+
+The option to access the MeshMap is available as the last option inside the sidebar of Meshery UI. Upon clicking the icon, the extension MeshMap will enable on your screen, and then you can choose the "design" option.
```
Comment (thisiskaransgit) [inline docs/pages/guides/meshmap.md:21]:
```suggestion
In the Dashboard, the MeshMap visualization tool is accessible via a sidebar option located at the bottom of the navigation menu. To access MeshMap, click on this option, and the MeshMap extension will activate on the screen. From there, you can select the "design" option.```

File: docs/assets/img/meshmap/meshmap-design-creation.png
Comment (Yana-Gupta) [reply docs/assets/img/meshmap/meshmap-design-creation.png:1]: okay @thisiskaransgit

**Comment by l5io** [comment]
🚀 Preview for commit 1014e744bebf5990976ebdab3bc6f6b7685081b8 at: https://64c7f375073d992f5aec7ac8--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e83011e8d9098be0bc793c66ba4ad639a4774952 at: https://64c7f6b79467062dbd58cccc--meshery-docs-preview.netlify.app

**Comment by Yana-Gupta** [comment]
@thisiskaransgit some test are failing. Could you please take a look about failing test and provide some insights.

**Comment by vishalvivekm** [comment]
@Yana-Gupta Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by thisiskaransgit** [comment]
@Yana-Gupta, No worries the workflow failure is not related to the PR.

**Comment by l5io** [comment]
🚀 Preview for commit abd5870df9a4df4c5ebde0a20a3b36f4d3c548e6 at: https://64cca210842a6d600297c3e5--meshery-docs-preview.netlify.app

**Comment by Yana-Gupta** [comment]
@thisiskaransgit please review

**Comment by l5io** [comment]
🚀 Preview for commit e31190ff23f65bc92d2dede71cf2d47afbdeb666 at: https://64d1f541fc96e83dce4c69b1--meshery-docs-preview.netlify.app

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 7d4dae4678bc1b7a3bd8223f4485f7dda80dcfd7 at: https://64d3c5d9d41d05008f880999--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 639f4103177f1ca35796ca6fdb31d6b345790618 at: https://64d3c72489a9ec073ebe5d39--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3b505a05ea902cabe5d765abcefcdf917125cdc9 at: https://64d65b2a0aeedc7cdee944b5--meshery-docs-preview.netlify.app

**Pull request closed without merging**